src/routes/quote.js

```diff
@@ -169,6 +169,10 @@ router.get(
         premiumInNXM: quote.premiumInNXM.toString(),
         premiumInAsset: quote.premiumInAsset.toString(),
         poolAllocationRequests: quote.poolAllocationRequests,
+        asset: {
+          symbol: ASSET_ID_MAPPING[coverAsset].symbol,
+          decimals: ASSET_ID_MAPPING[coverAsset].decimals,
+        },
```

Comment:
@MilGard91 I added as well `asset.symbol` for convenience. should we keep it or drop it?

Reply:
We should keep it